### ISSUE TITLE
Not worked for ruby 2.2.4.

### ISSUE THREAD

**Issue by zw963** [JoshCheek/seeing_is_believing#74]
Ruby 2.2.4, installed with rvm and a new gemset.

```sh
╭─ 23:51  zw963 ⮀ ~/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems ⮀ ➦ ruby-2.2.4@seeing_is_believing
 ╰─ $ seeing_is_believing 1.rb
program-without-annotations:1:1: error: cannot insert "$SiB.record_result(3, ("
program-without-annotations:1: 5.times do |i|
program-without-annotations:1:               
program-without-annotations:1:1: note: clobbered by: insert "begin; $SiB.number_of_captures = Float::INFINITY; "
program-without-annotations:1: 5.times do |i|
program-without-annotations:1:               
Parser::ClobberingError
Parser::Source::Rewriter detected clobbering

/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/parser-2.3.1.2/lib/parser/source/rewriter.rb:432:in `raise_clobber_error'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/parser-2.3.1.2/lib/parser/source/rewriter.rb:254:in `append'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/parser-2.3.1.2/lib/parser/source/rewriter.rb:67:in `insert_before'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/wrap_expressions.rb:36:in `block in call'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/wrap_expressions.rb:35:in `each'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/wrap_expressions.rb:35:in `call'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/wrap_expressions.rb:11:in `call'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing.rb:52:in `program_that_will_record_expressions'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing.rb:34:in `call'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing.rb:15:in `call'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/binary/add_annotations.rb:29:in `initialize'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/binary.rb:154:in `new'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/binary.rb:154:in `annotator'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/binary.rb:114:in `evaluate_program'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/lib/seeing_is_believing/binary.rb:45:in `call'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems/seeing_is_believing-2.2.0/bin/seeing_is_believing:6:in `<top (required)>'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/bin/seeing_is_believing:23:in `load'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/bin/seeing_is_believing:23:in `<main>'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/bin/ruby_executable_hooks:15:in `eval'
/home/zw963/.rvm/gems/ruby-2.2.4@seeing_is_believing/bin/ruby_executable_hooks:15:in `<main>'

 ╭─ 23:52  zw963 ⮀ ~/.rvm/gems/ruby-2.2.4@seeing_is_believing/gems ⮀ ➦ ruby-2.2.4@seeing_is_believing
 ╰─ $ 2  cat 1.rb
5.times do |i|
  i * 2
end

**Comment by JoshCheek** [comment]
Yeah, this is a failure on my part for not releasing v3 out of beta, which obviously should have happened already. For you, though, the solution is to do:

```sh
$ yes | gem uninstall seeing_is_believing
$ gem install seeing_is_believing --version 3.0.0.beta.7
```

And test it worked with

```sh
$ seeing_is_believing -e '1 + 1'
```

**Comment by zw963** [comment]
```sh
gem install seeing_is_believing --version 3.0.0.beta.7
```

is worked. Thanks.

**Comment by JoshCheek** [comment]
Closing b/c I released v3.0, so it should stop installing 2.x when you do a `gem install`

**Comment by zw963** [comment]
Cool!
